docs/src/install/gettingstarted.rst

```diff
@@ -0,0 +1,149 @@
+***************
+Getting Started
+***************
+
+Welcome! In this chapter, we give a quick overview of Oríon's main features and how it can help you
+streamline your machine learning workflow whether you are a researcher or engineer.
+
+Oríon is a black box function optimization library with a key focus on usability and integrability
+for its users. For example, as an machine learning engineer, you can integrate Oríon to your
+existing ML workflow to handle reliably the hyperparameter optimization and tuning phase. As a ML
+researcher, you can use Oríon to tune your models but also integrate your own algorithms to Oríon to
+serve as an efficient optimization engine and compare with other algorithms in the same context and
+conditions.
+
+Oríon is built to be non-intrusive, highly integrable, operate in parallel environments, and produce
+reproducible experiments. It takes only one line of code to start using it in your existing
+projects. It's natively asynchronous so it can perform efficiently on your laptop and in a computing
+farm with thousands of processors, without the need to configure a master and workers. Oríon
+supports the latest established hyperparameter algorithms out of the box, making it easy to switch
+between them or create benchmarks. It also supports a vast range of search spaces. Oríon uses a
+configuration agnostic approach where you can use any configuration file format you're comfortable
+with. The results are always stored in a database that can be either in-memory, local or remote.
+These results can be queried directly from the database itself or using convenient methods from
+Oríon's API. Additionally, the experiments are versioned -- think of it as a git for scientific
+experimentation -- enabling you to keep track of all your trials with their parameters. This
+guarantees that you can reproduce or trace back the steps in your work for free. Finally, a plugin
+system enables you to integrate and distribute your algorithms easily to other members of the
+community.
+
+Conversely, Oríon does not aim to be a machine learning framework or pipeline, or an automatic
+machine learning product. Oríon focuses essentially on black box optimization. However, we do
+encourage developers to integrate Oríon into that kind of systems as a component and we will do
+our best to help you if you're interested.
+
+Before continuing the overview, we assume that you have a basic understanding of machine learning
```

Comment:
This breaks the flow. I think the rest of this page is readable without making the installation and db setup.

Reply:
Where would you put it? If feel it's good to have it on this page, otherwise we never explicitly say to install Oríon and the database except in the README. Scikit-learn's getting started is using this model also.

Reply:
I don't think it is necessary to have installed Orion and setup the dataset to understand the overview, but I get your point about where should we point out doc for installation. Maybe changing a bit the wording would make it sound less mandatory. Something like the following.

> Before continuing the overview, we assume that you have a basic understanding of machine learning concepts. You may also want to install Oríon on your machine and configure it for a database before continuing. Please refer to our :doc:`installation instructions </install/core>` and :doc:`database
setup </install/database>`.

Reply:
I like it!